tests/pausing_test.go

```diff
@@ -198,15 +198,13 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		var vm *v1.VirtualMachine
 
 		BeforeEach(func() {
-			vmi := libvmifact.NewCirros(
+			vm = libvmi.NewVirtualMachine(libvmifact.NewCirros(
```

Comment:
nit: a0f8aae0cbb05013d636aed602076514df0e3f39 , suggestion for commit `Update tests to use correct RunStrategy instead of creating VMs stopped and then starting them. This simplifies these test cases.`

`Instead of Updating VM runStrategy in pausing_test.go create vm with
with the desired runStrategy.` is seems cryptic, at least for me.

Reply:
Done, thanks for the feedback.